cirq/api/google/v2/device.proto

```diff
@@ -46,13 +45,18 @@ message GateDefinition {
   // If unset or set to zero, any number of qubits is allowed.
   int32 number_of_qubits = 2;
 
-  // The name of the arguments that should be specified for 
+  // The name of the arguments that should be specified for
   // an operation of this gate
   repeated ArgDefinition valid_args = 3;
 
   // This defines the approximate amount of time for each gate,
   // specified as an integer number of picoseconds.
   int64 gate_duration_picos = 4;
+
+  // Valid targets that this gate can use.
+  // Values in this list correspond to the name of the TargetList
+  // If unset, all combinations using the number of qubits are allowed.
```

Comment:
maybe directly reference the field location?

Reply:
Changed to reference number_of_qubits.